src/main.rs

```diff
@@ -94,6 +100,26 @@ fn parse_line_referential() -> color_eyre::Result<HashMap<String, LineReference>
     Ok(line_referential)
 }
 
+fn parse_stop_referential() -> color_eyre::Result<HashMap<String, StopReference>> {
+    let ref_file = std::fs::File::open("static/data/arrets.csv")?;
```

Comment:
so, the gfts stop referential isn’t actually useful. We need an other one. In fact we don’t even use the gtfs. I coded it almost for nothing, but it will probably be useful at some point (and for the cache)

Reply:
Ok, I was indeed wondering why it was loaded :grin: